assets/src/infrastructure/ui/navigation/pagination/Pagination.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { CSSProperties } from 'react';
+import RcPagination from 'rc-pagination';
+import 'rc-pagination/assets/index.css';
+import defaultLocale from 'rc-pagination/lib/locale/en_US';
+
+import { PaginationProps } from './types';
+import PerPage from './PerPage';
+import defaultItemRender from './ItemRender';
+
+/* Temporary */
+const wrapperStyle: CSSProperties = {
+	display: 'flex',
+	flexDirection: 'row',
+	padding: '1em',
+};
+const paginationStyle: CSSProperties = {
+	display: 'flex',
+	marginRight: '1em',
+};
```

Comment:
I think it's ok to use `emotion` here

Reply:
Those pagination buttons may need more styling, so may be a separate styles file.